docs/topics/auth/default.txt

```diff
@@ -302,7 +302,7 @@ the user from the database. For example::
     from django.contrib.contenttypes.models import ContentType
     from django.shortcuts import get_object_or_404
 
-    from myapp.models import BlogPost
+    from .models import BlogPost
```

Comment:
There's no indication this example code lives in `myapp`, so I wouldn't use a relative import.

Reply:
Thanks Tim for your suggestions. It's my first PR and my first Django contribution so I truly value your advices.

I need clarification: if there's no an explicit indication above the code like `**Example views.py**::` should I default using an absolut import? Or is it preferable to understand the context before the code and choose the import on a case-by-case basis?

Reply:
If it looks like example code that doesn't live in a particular file, leave it as is.